spec/models/conference_spec.rb

```diff
@@ -778,6 +778,28 @@
     end
   end
 
+  describe '#scheduled_event_distribution' do
+    let(:conference) { create(:conference) }
+    let(:confirmed_unscheduled_event) { create(:event, program: conference.program, state: 'confirmed') }
+    let(:confirmed_scheduled_event) { create(:event_scheduled, program: conference.program) }
+
+    it '#scheduled_event_distribution does calculate correct values with events' do
+      confirmed_unscheduled_event
+      confirmed_scheduled_event
+      result = {} 
```

Comment:
remove the whitespace in this line to make Rubocop happy and add it to the commit you already have :wink: